src/modules/RootPart.rb

```diff
@@ -1805,8 +1805,8 @@ def MountPartitions(root_device_current)
         # in the target map of the storage module
         AddToTargetMap()
         if Yast2::FsSnapshot.configured?
-          snapshot = Yast2::FsSnapshot.create_pre("before upgrade")
-          Yast2::FsSnapshotStore.save("upgrade", snapshot.number)
+          snapshot = Yast2::FsSnapshot.create_pre("before update", cleanup: :number, important: true)
+          Yast2::FsSnapshotStore.save("update", snapshot.number)
```

Comment:
IIRC "update" means installing maintenance updates (patches), "upgrade" means product upgrade like @jreidinger mentioned. (That's **my** understanding, but maybe I'm wrong...)

I think we should ask @jsrain to get the authoritative answer...